README.md

```diff
@@ -19,31 +21,19 @@ and currency pairs.
   - Capacity to process the world’s cross-border payments volume
   - Easy access to liquidity through a competitive FX marketplace
 
-## Cross-border payments
-Ripple enables banks to settle cross-border payments in real-time, with
-end-to-end transparency, and at lower costs. Banks can provide liquidity
-for FX themselves or source it from third parties.
+## Global payments
+Ripple connects banks, payment providers, digital asset exchanges and corporates via RippleNet to provide one frictionless experience to send money globally.
```

Comment:
Given that the XRP Ledger isn't (currently) part of RippleNet, this feels off-topic for a README about `rippled` unless you have a proper lead-in. One could easily misinterpret that to think that the XRP Ledger and RippleNet were one and the same.